<launch>
  <arg name="use_rviz" default="true" />

  <node pkg="map_manager" type="nav_map_create" name="nav_map_create" output="screen">
    <param name="record" value="false" />
    <param name="leaf_size" value="0.1" />
    <param name="pcd_path" value="$(find map_manager)/pcd/" />
    <param name="visual_pcd_path" value="$(find map_manager)/pcd/terrain20250311_12_24.pcd" />
    <param name="use_sim_time" value="false" />
    <param name="obstacleHeightThre" value="0.15" />
    <param name="save_always" value="true" /> <!-- 一直保存 -->
  </node>

  <!-- 启动rviz -->
  <group if="$(arg use_rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rvizGA"
      args="-d $(find vehicle_simulator)/rviz/robot_simulator_Sentry.rviz" respawn="true" />
  </group>

</launch>